extensions/wikia/CommunityPage/templates/topContributors.mustache

```diff
@@ -19,7 +19,7 @@
 				<a href="{{profilePage}}">{{{avatar}}}</a>
 			</div>
 			<div class="contributor-details">
-				{{count}}. <a href="{{profilePage}}">{{userName}}</a>
+				{{count}}. <a data-tracking="top-contributors" href="{{profilePage}}">{{userName}}</a>
```

Comment:
Cool.
I would add user only to links that redirects to user profile page. What are your concerns?